kodilla-stream/src/main/java/com/kodilla/stream/world/World.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.stream.world;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class World {
+    private final List<Continent> continents = new ArrayList<>();
+
+    public BigDecimal getPeopleQuantity(){
+        BigDecimal peopleQuantity = new BigDecimal("0");
```

Comment:
tak metoda nigdy nie zwroci wartosci zadeklatowanej tutaj: new BigDecimal("0"). To jest do wyrzucenia